src/arviz_plots/plots/parallel_plot.py

```diff
@@ -0,0 +1,229 @@
+    if (
+        sample_stats is not None
+        and "diverging" in sample_stats.data_vars
+        and np.any(sample_stats.diverging)
+    ):
+        div_mask = sample_stats.diverging
+        if coords is not None:
+            div_mask = div_mask.sel(
+                {key: value for key, value in coords.items() if key in div_mask.dims}
+            )
+        if len(sample_dims) > 1:
+            div_mask = div_mask.stack(sample=sample_dims)
+    else:
+        div_mask = xr.zeros_like(data.coords[combined_dim], dtype=bool)
+    data = data.assign_coords(diverging=div_mask)
+    data = dataset_to_dataarray(
+        data, labeller=labeller, sample_dims=[combined_dim], label_type="vert"
+    )
+    if len(data.coords["label"].values) <= 1 or data.ndim != 2:
+        raise ValueError(
+            "Parallel plot requires at least two variables to plot. "
+            "Please provide more than one variable."
+        )
+
+    # get labels and x-values
+    x_labels = data.coords["label"].values
+    x_values = np.arange(len(x_labels))
+
+    new_sample_dims = [combined_dim, "label"]
+    plot_bknd = import_module(f".backend.{backend}", package="arviz_plots")
+    if plot_collection is None:
+        pc_kwargs.setdefault("cols", None)
+        pc_kwargs["figure_kwargs"] = pc_kwargs.get("figure_kwargs", {}).copy()
+        pc_kwargs = set_wrap_layout(pc_kwargs, plot_bknd, data)
+        pc_kwargs["aes"] = pc_kwargs.get("aes", {}).copy()
+        colors = plot_bknd.get_default_aes("color", 1, {})
+        pc_kwargs["aes"].setdefault("color", ["diverging"])
+        pc_kwargs["aes"].setdefault("alpha", ["diverging"])
+        pc_kwargs["color"] = pc_kwargs.get("color", [colors[0], "black"])
+        pc_kwargs["alpha"] = pc_kwargs.get("alpha", [0.05, 0.1])
+        plot_collection = PlotCollection.wrap(
+            data.to_dataset(),
+            backend=backend,
+            **pc_kwargs,
+        )
+
+    if aes_by_visuals is None:
+        aes_by_visuals = {}
+    else:
+        aes_by_visuals = aes_by_visuals.copy()
+    aes_by_visuals.setdefault("line", plot_collection.aes_set)
+
+    # plot lines
+    line_kwargs = copy(visuals.get("line", {}))
+    if line_kwargs is not False:
+        _, _, line_ignore = filter_aes(plot_collection, aes_by_visuals, "line", new_sample_dims)
+        plot_collection.map(
+            multiple_lines,
+            "line",
+            data=data,
+            x_dim="label",
+            xvalues=x_values,
+            ignore_aes=line_ignore,
+            **line_kwargs,
+        )
+
+    # x-axis label
+    xticks_kwargs = copy(visuals.get("xticks", {}))
+    if xticks_kwargs is not False:
+        _, _, xticks_ignore = filter_aes(plot_collection, aes_by_visuals, "xticks", new_sample_dims)
+        plot_collection.map(
+            set_xticks,
+            "xticks",
+            labels=x_labels,
+            values=x_values,
+            rotation=0,
```

Comment:
rotation should be added to `xticks_kwargs` as `setdefault` so users can override it